setup/flavors/compose/.env

```diff
@@ -0,0 +1,5 @@
+{% if bind4 %}
+EXTERNAL_IP={{ bind4 }}
+{% else %}
+EXTERNAL_IP=0.0.0.0
+{% endif %}
```

Comment:
We don't [officially support](https://mailu.io/1.7/compose/setup.html#bind-address) running on `0.0.0.0`. As such, I don't think we should provide this as a default:

> If the address is not configured directly (NAT) on any of the network interfaces or if you would simply like the server to listen on all interfaces, use 0.0.0.0 and ::. Note that running is this mode is not supported and can lead to [issues](https://github.com/Mailu/Mailu/issues/641).